### PR TITLE
Feature macosx

### PR DESCRIPTION
**Pull request by aspeteRakete**

### CODE REVIEW

**Comment by procedural** [comment]
Can I make few non-breaking adjustments to it after merge, @aspeteRakete?

**Comment by aspeteRakete** [comment]
of course

**Comment by procedural** [comment]
Merged! Thank you so much for this! I send you an invite.

Can I ask if someone will open an issue related to Mac version, can I ping you so you could help resolve it?

**Comment by procedural** [comment]
Also, I'm hosting precompiled versions of the libraries on release page, could you send me an archive with `lib-osx` folder for current master branch?

**Comment by procedural** [comment]
And I want to feature compile commands for OS X in readme file, can you write them for OS X, which project generator should user type? For example these are for Linux:
```bash
mkdir build
cd build
cmake .. -G Ninja
ninja
```

**Comment by procedural** [comment]
And what's you real name, by the way? :) Do you have a profile on [openFrameworks forum](http://forum.openframeworks.cc/)?

**Comment by aspeteRakete** [comment]
Project Generator:
I didn't specify any Generator which produces Unix Makeflies. They work and the output looks nice. I tested
``` 
-G Xcode
```
which works, but i got the feeling that compile times are very high.
Haven't tried out Ninja.

File: openFrameworks.cmake [hunk 1]
```diff
@@ -249,6 +325,10 @@ elseif(CMAKE_SYSTEM MATCHES Windows)
         list(APPEND OPENFRAMEWORKS_LIBRARIES
             ${OPENFRAMEWORKS_LIBS}
         )
+    elseif(CMAKE_SYSTEM MATCHES Darwin)
+        list(APPEND OPENFRAMEWORKS_LIBRARIES
+            ${OPENFRAMEWORKS_LIBS}
+        )
```
Comment (procedural) [inline openFrameworks.cmake]: I think this block will never be executed since it's under `elseif(CMAKE_SYSTEM MATCHES Windows)` statement

**Pull request merged**